packages/cli/test/fixtures/unit/create-git-meta/no-origin/config

```diff
@@ -0,0 +1,7 @@
+[core]
+        repositoryformatversion = 0
+        filemode = true
+        bare = false
+        logallrefupdates = true
+        ignorecase = true
+        precomposeunicode = true
```

Comment:
The tests will still work, but for consistency with real git directories, can we move this into `packages/cli/test/fixtures/unit/create-git-meta/no-origin/git/config`?